@here/harp-materials/lib/ShaderChunks/ExtrusionChunks.ts

```diff
@@ -50,10 +60,10 @@ vExtrusionAxis = vec4(normalMatrix * extrusionAxis.xyz, extrusionAxis.w);
 vec3 geometryNormal = normal;
 `,
     extrusion_pars_fragment: `
-uniform float extrusionRatio;
+varying float vExtrusionRatio;
```

Comment:
I don't clearly understand the reasons for putting extrusion ratio into 'varying' scope, it will be interpolated between every fragment during rasterization (intentionally?). Maybe different mechanism should be used for that anyway to maintain the same (non-interpolated) value - `uniform` or  `flat` qualifier that ensures that there’s no interpolation.

Reply:
As a matter of fact I feel that extrusion ratio should be pre-calculated (solved) per object and passed as uniform both in VS and FS. That seems to be most natural way.

Reply:
This patch in essence disables animation of extruded building for buildings with very small height.
There are two ways of implementing this:
 - creating two render batches - flat EB and hight EB, with different values of `extrusionRatio` unirorm
   - this can be implemented either in style - by defining two rules for `buildings` or ...
   - in "platform", i.e decoder emits two render batches for two types of buildings 
 - in shaders (current solution), so materials "just" work,

This patch choses:
 - "platform" approach, because MIN_BUILDING_HEIGHT and float precision issues are implementation detail (and limitation), so user shouldn't is not bothered with this and forced to write two rules for such simple feature.

 - shader approach to minimize change impact. Creating two render batches in decoder is much "more" code, shader approach introduces only minimal performance impact